mappings/net/minecraft/util/io/ReusableStream.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_3538 net/minecraft/util/ReusableStream
+CLASS net/minecraft/class_3538 net/minecraft/util/io/ReusableStream
```

Comment:
more like collection stuff

Reply:
Are you sure? I don't see a stream as collection stuff...

Reply:
Though when I read stream the first thing I think of is `InputStream`

Reply:
I couldn't see the source before, but now with CFR I can see it's a simple collection/stream wrapper so I've moved it to the right package.

The name needs to be revised though.